src/Steps/TMSSReqs.tsx

```diff
@@ -11,170 +11,170 @@ export const TMSSReqs = () => {
     {
```

Comment:
@RobertPorterfield - You updated the fields here, so GlobalStore.tsx needs updated too

Reply:
Good catch! I'll remove the definitions in GlobalStore.tsx and add a useEffect to initialize all the fields within this file